gui/default/syncthing/core/syncthingController.js

```diff
@@ -13,13 +13,21 @@ angular.module('syncthing.core')
         var restarting = false;
 
         function initController() {
-            LocaleService.autoConfigLocale();
-            setInterval($scope.refresh, 10000);
-            Events.start();
+            if (window.metadata) {
```

Comment:
Will do. The reason here is that the refresh just crashes and spams the error console since we're not authenticated yet. And like above, it might be appropriate to still initialize the locale service.